cms/djangoapps/contentstore/management/commands/tests/test_reindex_courses.py

```diff
@@ -47,7 +47,7 @@ def _build_calls(self, *courses):
 
     def test_given_no_arguments_raises_command_error(self):
         """ Test that raises CommandError for incorrect arguments """
-        with self.assertRaisesRegexp(CommandError, ".* requires one or more arguments.*"):
+        with self.assertRaisesRegexp(CommandError, ".* requires one or more *"):
```

Comment:
Was this an intentional change to the regexp, or a search/replace issue?  Same question for the next file.

Reply:
Yes these were intentional, the new messages look like "reindex_course requires one or more <course_id>s OR the --all or --setup flags."